src/index.html

```diff
@@ -1,17 +1,150 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <div class="input-row">
+          <label for="surname">Surname: </label>
+          <input type="text" id="surname"
+            name="surname"
+            autocomplete="off"
+            minlength="5"
+            maxlength="35">
```

Comment:
```suggestion
          <input
            type="text" 
            id="surname"
            name="surname"
            autocomplete="off"
            minlength="5"
            maxlength="35"
          >
```
Форматируй инпуты вот так

Reply:
got it!